src/officehours_api/tests.py

```diff
@@ -0,0 +1,146 @@
+from unittest import mock
+
+from django.test import TestCase, override_settings
```

Comment:
Nice to see unit tests!

Reply:
I think the backends would also be a high-value target for unit tests.